### PR TITLE
Add system logs page for loki istances and loki selector in system logs page

### PR DESCRIPTION
**Pull request by StefanoScarpetta**
This pull request adds a card in the settings page to navigate to the loki instances page (system logs page), where we can edit the retention_days of the loki instance, we can edit its label and we can delete the inactive ones. This pull request adds also a selector in the system logs page to choose which loki instance we want to search logs in, visible only when we have more than one loki instance.

Refs https://github.com/NethServer/dev/issues/6890

### CODE REVIEW

File: core/ui/src/components/system-logs/LogSearch.vue [hunk 1]
```diff
@@ -361,6 +408,10 @@ export default {
       type: Array,
       required: true,
     },
+    loki: {
```
Comment (andre8244) [inline core/ui/src/components/system-logs/LogSearch.vue:411]: Can we rename this variable with something more specific, like `lokiInstances`?

File: core/ui/src/components/system-logs/LokiCard.vue [hunk 2]
```diff
@@ -0,0 +1,184 @@
+<!--
+  Copyright (C) 2023 Nethesis S.r.l.
```
Comment (andre8244) [inline core/ui/src/components/system-logs/LokiCard.vue:2]: Let's move this component inside `src/components/settings`

File: core/ui/src/components/system-logs/LokiCard.vue [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+}
+
+.row {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 0.25rem;
+}
+
+.loki-card-skeleton {
```
Comment (andre8244) [inline core/ui/src/components/system-logs/LokiCard.vue:170]: This CSS class is not used and can be removed

File: core/ui/src/router/index.js [hunk 4]
```diff
@@ -124,6 +124,14 @@ const routes = [
         /* webpackChunkName: "node-firewall" */ "../views/NodeFirewall.vue"
       ),
   },
+  {
+    path: "/settings/system-logs",
+    name: "SettingsSystemLogs",
+    component: () =>
+      import(
+        /* webpackChunkName: "system-logs" */ "../views/settings/SettingsSystemLogs.vue"
```
Comment (andre8244) [inline core/ui/src/router/index.js:132]:
```suggestion
        /* webpackChunkName: "settings-system-logs" */ "../views/settings/SettingsSystemLogs.vue"
```

File: core/ui/src/views/SystemLogs.vue [hunk 5]
```diff
@@ -188,6 +192,7 @@ export default {
       },
       internalNodes: [],
       apps: [],
+      loki: [],
```
Comment (andre8244) [inline core/ui/src/views/SystemLogs.vue:195]: Can we rename this variable with something more specific, like `lokiInstances`?

File: core/ui/src/views/settings/SettingsSystemLogs.vue [hunk 6]
```diff
@@ -0,0 +1,536 @@
...
+    QueryParamService,
+    PageTitleService,
+  ],
+  pageTitle() {
+    return this.$t("system_logs.title");
+  },
+  data() {
+    return {
+      q: {},
+      loki: [],
```
Comment (andre8244) [inline core/ui/src/views/settings/SettingsSystemLogs.vue:285]: Can we rename this variable with something more specific, like `lokiInstances`?

File: core/ui/src/views/settings/SettingsSystemLogs.vue [hunk 7]
```diff
@@ -0,0 +1,536 @@
...
+      const isValidationOk = this.validateLokiInstanceRetention();
+      if (!isValidationOk) {
+        return;
+      }
+
+      this.error.setLokiInstanceRetention = "";
+      this.loading.setLokiInstanceRetention = true;
+      const taskAction = "configure-module";
+
+      // register to task completion
```
Comment (andre8244) [inline core/ui/src/views/settings/SettingsSystemLogs.vue:394]: Let's add the registration to task abortion

File: core/ui/src/views/settings/SettingsSystemLogs.vue [hunk 8]
```diff
@@ -0,0 +1,536 @@
...
+      const isValidationOk = this.validateLokiInstanceLabel();
+      if (!isValidationOk) {
+        return;
+      }
+
+      this.error.setLokiInstanceLabel = "";
+      this.loading.setLokiInstanceLabel = true;
+      const taskAction = "set-name";
+
+      // register to task completion
```
Comment (andre8244) [inline core/ui/src/views/settings/SettingsSystemLogs.vue:446]: Let's add the registration to task abortion

**Review by andre8244 — APPROVED** [review]

File: core/ui/public/i18n/en/translation.json [hunk 9]
```diff
@@ -930,7 +931,35 @@
...
-        "utc": "UTC"
+        "utc": "UTC",
+        "current_instance": "Active instance",
+        "loki_instance": "Loki instance",
+        "loki_instance_title": "Multiple Loki instance",
+        "loki_instance_description": "Logs are stored in more than one Loki instance. Choose one in the field below.",
+        "loki_tooltip": "Search logs from the active Loki instance, from a former leader node or from a Loki backup.",
+        "loki": {
+            "title": "Loki",
+            "tooltip": "Cluster nodes transmit their log streams to the active Loki instance, which operates on the leader node. When a leader node is replaced, the previous leader node retains its inactive Loki instance, allowing the Logs page to search through it until it is uninstalled.",
```
Comment (andre8244) [inline core/ui/public/i18n/en/translation.json:942]:
```suggestion
            "tooltip": "Cluster nodes transmit their log streams to the active Loki instance, which operates on the leader node. When a leader node is replaced, the previous leader node retains its inactive Loki instance, allowing the System logs page to search through it until it is uninstalled.",
```

**Review by andre8244 — APPROVED** [review]
Just a minor label fix

**Comment by Tbaile** [comment]
Hello!
Late at the party, I've got just a request:
Disable the `Set Label` and the `Delete` if node is offline, we can't tell them nothing due to missing connection :cry: 
image

**Review by andre8244 — APPROVED** [review]

**Pull request merged**